data/shwetasng.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "Shweta-Singh",
+  "type": "personal",
+  "bio": "**Sophomore pursuing Btech CSE**",
+
+  "tags": ["DevOps", "WebDev", "Open Source"],
+
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
```

Add the Twitter username, if you don't have remove that part.

```suggestion
    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
```